File: 32 - Convert Snake Case to Camel Case/solution.js
function toCamelCase(str) {
  return str
    .split("_")
    .map((word, index) => {
      return index === 0
        ? word.toLowerCase()
        : word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
    })
    .join("");
}

// Test Cases
console.log(toCamelCase("snake_case")); // SnakeCase
console.log(toCamelCase("snake_case_123")); // SnakeCase123

File: 32 - Convert Snake Case to Camel Case/solution.py
def to_camel_case(str):
    parts = str.split("_")
    camel_case = parts[0].lower()

    for part in parts[1:]:
        camel_case += part.capitalize()

    return camel_case

# Test Cases
print(to_camel_case("snake_case"))  # SnakeCase